omod/src/test/java/org/openmrs/module/coreapps/fragment/controller/visit/RetrospectiveVisitFragmentControllerTest.java

```diff
@@ -65,14 +65,9 @@ public void test_shouldCreateNewRetrospectiveVisit() throws Exception {
         Date startDate = new DateTime(2012, 1, 1, 12, 12, 12).toDate();
         Date stopDate = new DateTime(2012, 1, 2, 13,13, 13).toDate();
 
-        // should round to the time components to the start and end of the days, respectively
-        Date expectedStartDate = new DateTime(2012, 1, 1, 0, 0, 0, 0).toDate();
```

Comment:
Because we want to preserve existing behavior when not using timezones, shouldn't we leave existing tests intact and only create new ones to deal with cases where the timezone global property is set?

Reply:
We changed the way that retrospective visit put the time, as i explained above :

> We changed the way that we format the start/end date, but in the end, everything should be the same as before
> Before we sent 2 dates from the client, and then we format on the server side to be startDate 00:00:00 and endDate 23:59:59:59.
> Now we are formatting the 00:00:00 and 23:59:59 on the client side. here, this way, if we are using timezone, we can send the date correctly formatted with timezone, if we are not using timezones, we will send the information without timezone, here

Because of that, some test need to be adapted, because now, we no longer add the 00:00:00 and 23:59:59 on the server side.

Reply:
If the time zone global property is not set, or set to false, i would expect this test to pass to preserve the old behaviour.